app/views/admin/school_groups/users/show.html.erb

```diff
@@ -23,6 +23,7 @@
         <th>Confirmed?</th>
         <th>Last signed in</th>
         <th>Language</th>
+        <th>Locked?</th>
```

Comment:
The locked status and buttons should only be visible to admins, and not group admins.